new.css

```diff
@@ -78,15 +78,15 @@ body {
 	margin: 0 auto;
 	max-width: 750px;
 	padding: 2rem;
-	border-radius: 6px;
+	/* removed border-radius because it makes no sense */
 	overflow-x: hidden;
 	word-break: break-word;
 	overflow-wrap: break-word;
 	background: var(--nc-bg-1);
 
 	/* Main body text */
 	color: var(--nc-tx-2);
-	font-size: 1.03rem;
+	font-size: 1rem; /* make fonts more crispy to 16px */
```

Comment:
Just saw that you wrote the following:

> added font-size: 1rem; to text input fields to prevent zooming in mobile browsers

Maybe update the comment to explain how and why this works? Maybe a link to an article? That way we avoid that people like me will make a PR to remove it again ;)